test cases/fortran/7 generated/meson.build

```diff
@@ -1,18 +1,19 @@
 # Tests whether fortran sources files created during configuration are properly
 # scanned for dependency information
 
-project('generated', 'fortran')
+project('generated', 'fortran',
+  default_options : ['default_library=static'])
 
 conf_data = configuration_data()
 conf_data.set('ONE', 1)
 conf_data.set('TWO', 2)
 conf_data.set('THREE', 3)
 
 outfile = configure_file(
-      input : 'mod3.fpp', output : 'mod3.f90', configuration : conf_data)
+  input : 'mod3.fpp', output : 'mod3.f90', configuration : conf_data)
 # Manually build absolute path to source file to test
 # https://github.com/mesonbuild/meson/issues/7265
-three = library('mod3', meson.current_build_dir() / 'mod3.f90')
+three = library('mod3', outfile)
```

Comment:
This change should not be here, right? https://github.com/mesonbuild/meson/pull/7266#issuecomment-643917454